Kitodo-DataManagement/src/main/java/org/kitodo/api/ugh/UghImplementation.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.ugh;
+
+import ugh.LegacyUghImplementation;
+
+public class UghImplementation {
+    public static final FactoryInterface INSTANCE = new LegacyUghImplementation();
```

Comment:
This should work, but I don't know if this is thread safe or causing other problems.